### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by zinfin**
Replaced with content written for developer's site.

### CODE REVIEW

**Comment by doneill** [comment]
@zinfin Let's move this to the [apps README](https://github.com/Esri/maps-app-android/blob/maps-app-doc/maps-app/README.md) and keep this README more about setting up the developer environment, then we can link to this doc for app context.

File: README.md [hunk 1]
```diff
@@ -1,147 +1,3 @@
...
-
-## Features
-* Dynamically switch basemaps
-* Place search
-* Routing
-* Geocode addresses
-* Reverse geocode
-* Sign in to ArcGIS account
 
 ## Development Instructions
```
Comment (doneill) [inline README.md]: Add an into above this for context

# Overview
The maps-app is your organizations mapping application that allows for switching basemaps, searching, routing, and authenticating against your ArcGIS organization for custom basemaps.

**Comment by doneill** [comment]
@zinfin We should discuss the usage of redirect uri as well as we should not require one if the user is logging into AGOL.  We have an issue to support this in the API but until then we should simplify in this app.

**Comment by zinfin** [comment]
@doneill In the top level README in this branch https://github.com/Esri/maps-app-android/tree/build_1517

**Comment by zinfin** [comment]
@doneill We can certainly revert to HTTP auth, thus removing any dependencies on a client id or a redirect uri, but it seems like showing how to do this would be important for non AGOL access (on premise servers)?  It's been difficult to find good documentation on how to do this correctly.

**Comment by doneill** [comment]
@zinfin I am not suggesting reverting back to HTTP, what I am saying is the app should default to `https://www.arcgis.com` which uses a default redirect uri so we should not require developers to create a redirect when working with `https://www.arcgis.com`.  All a developer needs is the Client ID in this case.  This will make is much easier for devs who are evaluating the app, devs who want to use a custom portal url need to create the redirect so we would document that case specifically.

**Comment by doneill** [comment]
@zinfin Refer to the [create save map](https://github.com/Esri/arcgis-runtime-samples-android/blob/v100/create-save-map/README.md) to see how our sample uses default redirect uri with default AGOL portal.

**Comment by doneill** [comment]
@zinfin Should we ultimately close this PR in favor of #147?

**Comment by zinfin** [comment]
That makes sense.  I didn’t know we didn’t need to set the redirect if the url is https://www.arcgis.com.  My assumption had been that when you registered an app you always needed both a client id and a re-direct uri, based on this doc https://developers.arcgis.com/authentication/mobile-and-native-user-logins/.

**Comment by doneill** [comment]
@zinfin Yes it can be confusing that is why I think we should be explicit in apps that show this pattern.  It is familiar to JS API devs as the API defaults this way, I am trying to get Runtime API's to do the same thing which should simplify the developer pattern. Let me know if you need assistance in documenting the approach.

**Comment by doneill** [comment]
@zinfin Good to merge

**Pull request merged**